plugins/bookmarks.js

```diff
@@ -992,14 +992,16 @@ window.plugin.bookmarks.loadStorageBox = function() {
       if(fullUpdated) {
         window.plugin.bookmarks.refreshBkmrks();
         window.plugin.bookmarks.resetAllStars();
-        console.log('BOOKMARKS: synchronized all after offline');
+        window.runHooks('pluginBkmrksSyncEnd', {"target": "all", "action": "sync"});
+        console.log('BOOKMARKS: synchronized all from drive after offline');
```

Comment:
Do we need this?

Reply:
yes, otherwise Book-Markers would not update on map
console.log: me I like it , you do not I guess.